Rubberduck.Parsing/Preprocessing/EmptyValue.cs

```diff
@@ -8,65 +8,20 @@ public sealed class EmptyValue : IValue
     {
         public static readonly EmptyValue Value = new EmptyValue();
 
-        public ValueType ValueType
-        {
-            get
-            {
-                return ValueType.Empty;
-            }
-        }
+        public ValueType ValueType => ValueType.Empty;
```

Comment:
Should be `ValueType = ValueType.Empty;`

Reply:
Doing so causes it to the error `Interface member ... is not implemented`.